pandas/tests/frame/test_subclass.py

```diff
@@ -13,6 +13,16 @@
 import pandas._testing as tm
 
 
+@pytest.fixture()
```

Comment:
whats wrong wiht tm.SubclassedDataFrame?

Reply:
Good question, the reason is that `convert_dtypes` uses `pd.concat` to merge the series for each column back together. Internally, this uses `_constructor_expanddim` of the Series for the first column - so we need `_constructor_sliced` to permit `pd.Series` being returned to see this issue.

(This pattern is used in Geopandas, where a GeoDataFrame contains a GeoSeries which has geometry data in it, but also numeric/ string attributes stored in ordinary Series - unliked tm.SubclassedDataFrame where all columns are SubclassedSeries. This is referenced in the linked issue but there's some more discussion on this at https://github.com/geopandas/geopandas/issues/1870)